src/videouplinkbandwidthpolicy/VideoUplinkBandwidthPolicy.ts

```diff
@@ -57,4 +58,25 @@ export default interface VideoUplinkBandwidthPolicy {
    * Returns the selected [[MediaTrackConstraints]] to update
    */
   chooseMediaTrackConstraints(): MediaTrackConstraints;
+
+  /**
+   * Set a reference to the current transceiver controller.
+   * This should be called whenever a transceiver controller is available when the call starts.
+   * Calls this with a TransceiverController when the meeting starts, and calls it with undefined when the meeting ends.
```

Comment:
Calls => call, in both places. But note that you're writing for two audiences: people implementing a controller that will call this ("… call this when the meeting starts") and people implementing a policy ("this will be called by the audio video controller when the meeting starts"). Right now your wording falls between those two — you say "should be called" as if you're writing for the former, and then "will be repeated" as if you're writing for the latter.

Reply:
Updated